metadata.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "bbriggs-rocketchat",
```

Comment:
The paste buffer strikes again!

-- Briggs

> On Jun 11, 2016, at 6:50 PM, Alex Fisher <notifications@github.com> wrote:
> 
> In metadata.json:
> 
> > @@ -0,0 +1,56 @@
> > +{
> > +  "name": "bbriggs-rocketchat",
> or...
> puppet-minecraft ;)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>